card/lib/generators/card.rb

```diff
@@ -0,0 +1,43 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Note new location for generators.  This seems to be where rails wants them so they are found when the generator commands are loaded.